migrations/versions/90f976272da9_.py

```diff
@@ -0,0 +1,24 @@
+"""Rename node.connection_config
+
+Revision ID: 90f976272da9
+Revises: 3d7bcbbf6cc4
+Create Date: 2018-01-08 10:48:14.069122
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '90f976272da9'
+down_revision = '3d7bcbbf6cc4'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.alter_column('node', 'connection_config', existing_type=sa.Text, new_column_name='proposals')
```

Comment:
Lets not alter `connection_config`, but add new column `proposal`.
And from this point of time new structure version will go to new column.

Reply:
Done.